scripts/resetdb.sh

```diff
@@ -53,7 +53,7 @@ main() {
       echo "Resetting DB..."
       mysql "${FLAGS[@]}" -e "DROP DATABASE IF EXISTS ${DB_NAME};"
       mysql "${FLAGS[@]}" -e "CREATE DATABASE ${DB_NAME};"
-      mysql "${FLAGS[@]}" -e "GRANT ALL ON ${DB_NAME}.* TO '${DB_NAME}'@'localhost' IDENTIFIED BY 'zaphod';"
```

Comment:
Probably could specify a net-mask here to make it a bit narrower.

Reply:
This script was originally written for running tests so wasn't particularly strict.

Reply:
SGTM, leaving as is.